articles/native-platforms/wpf-winforms/01-login.md

```diff
@@ -78,6 +78,6 @@ The `Auth0User` has the following properties:
 
 ## Download the samples
 
-Browse the samples on GitHub from [here](https://github.com/auth0/Auth0.WinformsWPF/tree/master/samples).
```

Comment:
@mehreencs87 Please remove this paragraph and instead use the seed template for this. 
Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects
Thanks.